README.rst

```diff
@@ -4,14 +4,71 @@
 .. image:: https://travis-ci.org/camptocamp/marabunta.svg?branch=master
     :target: https://travis-ci.org/camptocamp/marabunta
 
-Marabunta is a name given to the migration of the legionary ants or to the ants
-themselves. Restless, they eat and digest everything in their way.
+*Marabunta is a name given to the migration of the legionary ants or to the ants
+themselves. Restless, they eat and digest everything in their way.*
 
-This tool aims to run migrations for Odoo versions as efficiently as a
-Marabunta migration.
+Marabunta is used to provide an easy way to create Updates for Odoo fast and run easily. It also allows to differentiate between different environment to provide for instance demodata.
+
+
+Usage
+=====
+After installing marabunta, it will be available as a console command. To run properly it requires a migration file (which defines what has to updated/executed) and odoos connection parameters (view options in the options section.
+
+At each run marabunta verifies the versions from the migration file and and processes new ones.
+It is very much recommended to configure it, so that marabunta is ran automatically if odoo is started.
+For instance adding it to your docker entrypoint.
+
+Features
+========
+
+* backup: Marabunta allows for a backup command to be executed before the migration.
+* addon upgrades: Marabunta is able to install or upgrade odoo addons.
+* addon uninstall: Addons which were previously installed can be removed with marabunta.
```

Comment:
is that true that Marabunta is capable of removing addons w/o anthem?

Reply:
@arkostyuk you're right. It is in the example file and has functions to handle it but they raise an error that it is not possible.